adafruit_requests.py

```diff
@@ -652,16 +710,18 @@ def connect(self, address):
 
 
 class _FakeSSLContext:
-    def __init__(self, iface):
+    def __init__(self, iface: InterfaceType) -> None:
         self._iface = iface
 
-    def wrap_socket(self, socket, server_hostname=None):
+    def wrap_socket(
+        self, socket: SocketType, server_hostname: Optional[str] = None
+    ) -> _FakeSSLSocket:
         """Return the same socket"""
         # pylint: disable=unused-argument
         return _FakeSSLSocket(socket, self._iface.TLS_MODE)
 
 
-def set_socket(sock, iface=None):
+def set_socket(sock, iface: Optional[InterfaceType] = None) -> None:
```

Comment:
What is the type of `sock`?

Reply:
`sock` is actually a module, either CircuitPython's `socketpool` or CPython's `socket` module.  Basically, it's any module which implements the `socket` class.